sdk/internal/recording/recording.go

```diff
@@ -624,11 +626,37 @@ func GetRecordingId(t *testing.T) string {
 }
 
 func GetRecordMode() string {
+	if !(recordMode == RecordingMode || recordMode == PlaybackMode || recordMode == LiveMode) {
+		recordMode = PlaybackMode
+	}
 	return recordMode
 }
 
+func findProxyCertLocation() (string, error) {
+	fileLocation, ok := os.LookupEnv("PROXY_CERT")
+	if ok {
+		return fileLocation, nil
+	}
+
+	// Couldn't be found, lets try to find it from the GOPATH location
+	var err error
+	goPath, ok := os.LookupEnv("GOPATH")
+	if !ok {
+		goPath, err = os.UserHomeDir()
```

Comment:
Is this a conventional GOPATH? If not, I think it's okay to require devs to set GOPATH or PROXY_CERT.

Reply:
Yes, if it is not set Go defaults the home directory: https://golang.org/doc/gopath_code#GOPATH